protocols/bgp/server/fsm_address_family.go

```diff
@@ -87,13 +103,11 @@ func (f *fsmAddressFamily) processUpdate(u *packet.BGPUpdate) {
 		return
 	}
 
-	if f.multiProtocol {
```

Comment:
That's kind of a logic change. Are we sure that IPv4 updates will never code as MP updates, ever?

Reply:
To be approved by Dan.

Reply:
If we announce MP cap for IPv4 (which is possible by setting `Peer.AdvertiseIPv4MultiProtocol` to `true` and the other side does so too, we are supposed to send/receive updates for IPv4 as MP_REACH.

Reply:
That's O.K. for me and the RFC (4760)